smartsim/_core/control/job.py

```diff
@@ -299,7 +299,7 @@ def error_report(self) -> str:
         warning += f"Job status at failure: {self.status} \n"
         warning += f"Launcher status at failure: {self.raw_status} \n"
         warning += f"Job returncode: {self.returncode} \n"
-        warning += f"Error and output file located at: {self.entity.path}"
+        # warning += f"Error and output file located at: {self.job.path}"
```

Comment:
Is this commented out because it wrong or because it just won't work?

Reply:
Path is no longer on entity, for a second I thought I could access off of Job but no. I commented it out and left it incase we want to replace in the future, it will not be forgotten